test/test_operations.py

```diff
@@ -223,10 +223,12 @@ def forward(self, x):
     x = self.fc2(x)
     return F.log_softmax(x, dim=1)
 
-
+@unittest.skipIf(xr.device_type() == 'CUDA',
+    'Parallelism for DataParallel uses multi-threads. But cuda assumes one GPU device per process instead of relying on threads.')
```

Comment:
Not for this PR, but IMO we should just delete these tests. Do we support `DataParallel` anymore @JackCaoG?